com.github.vkohaupt.vokoscreenNG.yml

```
@@ -0,0 +1,36 @@
+app-id: com.github.vkohaupt.vokoscreenNG
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: vokoscreenNG
+rename-desktop-file: vokoscreenNG.desktop
+rename-appdata-file: vokoscreenNG.appdata.xml
+rename-icon: vokoscreenNG
+finish-args:
+  - --device=all
+  - --device=dri
```

```suggestion
  - --device=all
```

dri is include in all.